packages/client/schemaExtensions/clientSchema.graphql

```diff
@@ -44,6 +44,7 @@ extend type RetrospectiveMeeting {
 extend type TeamPromptMeeting {
   localStageId: String
   isRightDrawerOpen: Boolean!
+  showWorkSidebar: Boolean!
```

Comment:
+1 can we reduce number of flags to just 2? ie do the following
1. use `localStageId` to control if discussion drawer is showing (if it's set, show, if it's not do not show it)
2. use `isWorkSidebarShowing` to control if task drawer is showing

is there anything I'm missing that'd make this solution no viable? if not, it _should_ simplify the render conditionals , make it easier to understand and more explicit

Reply:
Not quite - on smaller screens it's possible to open the drawer by swiping in from the right (e.g. open a discussion thread, close the drawer, then swipe the drawer back in to open the discussion again), so we need both a) the state describing the drawer contents, and b) whether the drawer is actually open right now.